spec/helpers/address_helper_spec.rb

```diff
@@ -0,0 +1,119 @@
+require "spec_helper"
+
+RSpec.describe AddressHelper, type: :helper do
+  context "allow net connections" do
+    before do
+      WebMock.allow_net_connect!
+      VCR.turn_on!
+    end
+
+    after do
+      WebMock.disable_net_connect!(allow_localhost: true)
+      VCR.turn_off!
+    end
+
+    context "200 - valid postcode" do
+      it "returns all UPRN's for a valid estate block of self-contained flats" do
+        VCR.use_cassette "address/200_multi_response" do
+          response = JSON.parse(helper.postcode_lookup("SW1P2LQ"))
+
+          expect(response["code"]).to be(200)
+          expect(response["results"].count).to be(14)
+        end
+      end
+
+      it "returns a single UPRN for an address with it's own postcode" do
+        VCR.use_cassette "address/200_single_response" do
+          response = JSON.parse(helper.postcode_lookup("SW1A2AA"))
```

Comment:
If we return a hash in https://github.com/alphagov/govuk-coronavirus-vulnerable-people-form/pull/293/files#diff-d0becce4dbc49e088754c6a09e42ca4eR19 rather than a JSON string, then we won't need to parse the JSON on each request.

Reply:
Thanks @brucebolt - updated as per your suggestion.